app/src/main/java/com/codesoom/assignment/controllers/UserController.java

```diff
@@ -0,0 +1,56 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.UserService;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/users")
+@CrossOrigin
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    /**
+     *  userData에 user을 생성하고 리턴한다.
+     *
+     * @param userData 생성된 user 정보
+     * @return 생성된 user
+     */
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public User create(@RequestBody @Valid UserData userData) {
+        return userService.createUser(userData);
+    }
+
+    /**
+     * id에 해당되는 user을 수정하여 userData 리턴한다.
+     *
+     * @param id 수정할 user의 id
+     * @param userData 수정한 user 정보
+     * @return 수정된 user 정보
+     */
+    @PatchMapping("/{id}")
+    public User update(
+            @PathVariable Long id,
+            @RequestBody @Valid UserData userData) {
+        return userService.updateUser(id, userData);
+    }
+
+    /**
+     * id에 해당되는 user을 삭제한다.
+     * @param id 삭제할 user id
+     */
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void destroy(@PathVariable Long id) {
```

Comment:
이런 경우엔 메소드 시그니처에도 약간 더 신경써주는 센스를 발휘할 수 있겠죠.
```suggestion
    public void destroy(@PathVariable Long userId) {
```

Reply:
다른 메소드에도 적용시키겠습니다